crypto3/libs/multiprecision/include/nil/crypto3/multiprecision/integer_ops_powm.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <limits>
+#include <type_traits>
+
+#include "nil/crypto3/multiprecision/big_uint_impl.hpp"
+#include "nil/crypto3/multiprecision/ops/powm.hpp"
+
+namespace nil::crypto3::multiprecision {
+    template<typename T1, typename T2, typename T3,
+             std::enable_if_t<std::numeric_limits<std::decay_t<T1>>::is_integer &&
+                                  std::numeric_limits<std::decay_t<T2>>::is_integer &&
+                                  std::is_integral_v<T3>,
+                              int> = 0>
+    constexpr T3 powm(T1 &&b, T2 &&e, T3 m) {
```

Comment:
Maybe not in this PR, but we can remove all these functions, and make them members of some classes.

Reply:
This should work for built-in types as well